connect/src/com/telenor/connect/ui/ConnectLoginButton.java

```diff
@@ -107,6 +107,11 @@ public void onClick(View v) {
                 parameters.put("scope", TextUtils.join(" ", getLoginScopeTokens()));
             }
 
+            final String mccMncLoginHint = ConnectSdk.getMccMncLoginHint();
```

Comment:
For more context I guess you can see the design doc in Confluence? But in short it's the MCCMNC pair sent as a login_hint, so that HE service could pick the right BU data reader from it.